common/blocks.py

```diff
@@ -500,10 +500,17 @@ class VerticalBarChart(blocks.StructBlock):
         'e.g., "Bar chart showing a decreasing number of assaults over the '
         'course of 2023."',
     )
+    group_by = blocks.ChoiceBlock(
+        label='Group Incidents By',
+        required=False,
+        choices=charts.IncidentBranches.choices,
+        default=None,
+    )
```

Comment:
Might be nice here to add some help text explaining how this field changes how the chart is displayed. Though I'm not immediately coming up with a concise way of doing that.

Reply:
"If selected, will create a stacked bar chart colored by grouping" or something along those lines?

Reply:
@harrislapiroff how's this? 

"If selected, turns this vertical bar chart into a stacked bar chart with incidents grouped by the selected classification."